moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectECapris.js

```diff
@@ -182,7 +180,7 @@ const ProjectSummaryProjectECapris = ({
         {!editMode && (
           <ProjectSummaryLabel
             text={
-              (isValidNumber(eCapris) && (
```

Comment:
was this check before to remove invalid ecapris values from being shown? and now all the ecapris are valid?

Reply:
i'm really not sure! maybe it was preventing `undefined` from being rendered?

Reply:
to answer your question—all ecapris values were always valid in the sense that nothing would break if they were rendered. we actually have a handful of ids that are not "valid" according to the new regex implemented here. so those will still be rendered but the user would need to edit the value in order to save it again.